src/components/Form/FormFields/FormField.tsx

```diff
@@ -54,6 +55,16 @@ const FormField = ({
   field?: FormFieldBaseProps<any>;
   children: React.ReactNode;
 }) => {
+  const [phhelp, setPhhelp] = useState(false);
+
+  useEffect(() => {
+    if ((field && field?.help == undefined) || (field && !("help" in field))) {
+      setPhhelp(true);
+    } else if (field?.help == false) {
+      setPhhelp(false);
+    }
+  }, []);
+
```

Comment:
I don't see a reason why we need a useState and a useEffect to show help in sufix. Can't we keep it how it was before? And pass a prop to the `PhoneNumberFormField` directly instead?

Reply:
@rithviknishad ,no it will not work please try to understand,passing prop from phnumberfield will not show help icon fir other components ,for serchfield it will be false

Reply:
@i0am0arunava have you read about initalstate prop for a state? 

https://react.dev/reference/react/useState#parameters